.github/workflows/create-release-draft.yml

```diff
@@ -186,6 +184,73 @@ jobs:
         run: |
           make clean-docker-${{ matrix.chain }}
 
+  ## check extrinsic ##
+  extrinsic-ordering-check-from-bin:
+    runs-on: self-hosted
+    needs: build-docker
+    strategy:
+      matrix:
+        chain: [rococo, litmus, litentry]
+        include:
+          - chain: rococo
+            ref_url: wss://rpc.rococo-parachain-sg.litentry.io
+          - chain: litmus
+            ref_url: wss://rpc.litmus-parachain.litentry.io
+          - chain: litentry
+            ref_url: wss://rpc.litentry-parachain.litentry.io
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.RELEASE_TAG }}
+      - name: Prepare output and compare the metadata
+        timeout-minutes: 3
+        run: |
+          RELEASE_TAG=${{ env.RELEASE_TAG }}
+          PARACHAIN_NAME=local-parachain
+          BASE_URL=ws://127.0.0.1:9944
+          chain=${{ matrix.chain }}
+          REF_URL=${{ matrix.ref_url }}
+          echo "Metadata comparison:" > output-$chain.txt
+          echo "Date: $(date)" >> output-$chain.txt
+          echo "Base: $BASE_URL" >> output-$chain.txt
+          echo "Reference: $REF_URL" >> output-$chain.txt
+          echo "Target version: $VERSION" >> output-$chain.txt
```

Comment:
I think `VERSION` is always empty here? That's also what I saw in the example issue.

Reply:
change to `RELEASE_TAG`